src/release/breaking-changes/ignoringsemantics-migration.md

```diff
@@ -41,10 +56,22 @@ ExcludeSemantics(
     child: const PlaceHolder(),
   ),
 );
+
+ExcludeSemantics(
+  child: AbsorbPointer(
+    child: const PlaceHolder(),
+  ),
+);
+
+SliverIgnorePointer(
+  child: ExcludeSemantics(
+    child: const PlaceHolder(),
+  ),
+);
 ```
 
-The behavior of setting `ignoringSemantics` to false is no longer supported.
-Consider creating your own custom widget.
+The behaviors of setting `ignoringSemantics` to false are no longer supported.
+Consider creating your own custom widgets.
```

Comment:
```suggestion
Setting `ignoringSemantics` to false is no longer supported.
Consider creating your own custom widgets.
```
Why would you need to create your own custom widgets? This needs a tad more explanation.

Reply:
I am not sure why people would want to have the exact behavior of setting `ignoringSemantics` to true. so I only provide a example custom widgets they can use if they really really want to have the exact behavior